go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/gobwas/glob v0.2.3
 	github.com/gofrs/flock v0.8.1
 	github.com/gosuri/uitable v0.0.4
+	github.com/hashicorp/go-multierror v1.1.1
```

Comment:
With the built-in multi-error support in Go 1.20, it might be an idea to put a `// TODO` somewhere once the version in `go.mod` allows to do so. Which won't happen with #11896, but support for `1.19` should end in [~5 months](https://endoflife.date/go).

Reply:
I don't think it's necessary to introduce a new library, we can use`k8s.io/apimachinery/pkg/util/errors.NewAggregate([]error)` to aggregate errors, be consistent with the behavior (incorrect format) of Kubernetes incidentally

Reply:
What's happening here is fine for now.